packages/tracing/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@sentry/tracing",
+  "version": "5.19.2",
+  "description": "Extensions for Sentry AM",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/tracing",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=6"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "5.19.2",
```

Comment:
Why do we include `@sentry/browser` in tracing? Shouldn't we remove.

Reply:
Yes, we should remove it if possible.